packages/core/src/utilities.ts

```diff
@@ -63,3 +63,16 @@ export function applyStyles(
 
   return flattenedStyles;
 }
+
+export function getRealValue(value: any) {
+  // console.log("getRealValue typeof", typeof value);
+  // console.log("getRealValue value", value);
```

Comment:
I think you can remove these comments?

Reply:
@peter @dhruska You can also pass in an undefined and get back an undefined, which it now accepts as an optional type arg.  I used Peter's notes and added an error if the arg is not type undefined, string, or number :)